src/components/AssetImage/AssetImage.tsx

```diff
@@ -0,0 +1,29 @@
+import classNames from 'classnames'
+import React from 'react'
+import { Rarity } from '@dcl/schemas/dist/dapps/rarity'
+import './AssetImage.css'
+
+export type AssetImageProps = {
+  name: string
+  rarity: Rarity
+  src: string
+  className?: string
+}
+
+export const AssetImage = (props: AssetImageProps) => {
+  const { name, rarity, src, className } = props
+
+  const [light, dark] = Rarity.getGradient(rarity)
+  const backgroundImage = `radial-gradient(${light}, ${dark})`
+
+  return (
+    <div
+      className={classNames('AssetImage', className ?? '')}
```

Comment:
The `classNames` dependency automatically deletes the classes that are not truthy, so there's no need to check for it being undefined or null:
```suggestion
      className={classNames('AssetImage', className)}
```